docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md

```diff
@@ -63,3 +63,5 @@ The Stripe App has limited or no support for the following features:
 - Payment Authorization & Capture: The current logic works with separate authorization & capture. Hence, all payment methods go through this transition.
 - Payment Authorization Timeout: There is currently a one-day timeout for authorizing payments. This means that payment methods such as bank transfers, which are not authorized within this timeframe, will experience a timeout. We recommend extending the timeout from one day to seven days. 
 - Multi-Capture: Partial capture of payment for order items.
+- Payments should always be canceled from Spryker and not from Stripe.
+- Payments can’t be partially cancelled. Reason: we create one payment intent per order and it can either be authorized or fully cancelled .
```

Comment:
```suggestion
- Payments can’t be partially cancelled. Reason: we create one payment intent per order and it can either be authorized or fully cancelled.
```